src/Telemetry/Library/TelemetryManager.cs

```diff
@@ -66,14 +66,16 @@ public static class TelemetryManager
         public static TelemetryManagerConfig Configuration => Instance.Configuration;
 
         /// <summary>
-        /// If this value is true, no data will be collected or sent.
-        /// Every public method of this class check this variable first and if it's true the method
+        /// If this value is false, no data will be collected or sent.
+        /// Every public method of this class check this variable first and if it's false the method
         /// immediately returns without doing anything.
         /// This property defaults to "false".
-        /// This property is automatically set to true during initialization if the environment variable
-        /// named in Configuration.HostingEnvironmentVariableName has a value of "1".
+        /// This property is automatically set during initialization based on the
+        /// Configuration.DefaultTelemetryConsent and based on the environment variables
+        /// named in Configuration.TelemetryOptOutVariableName and
+        /// Configuration.TelemetryOptInVariableName.
         /// </summary>
-        public static bool TelemetryOptOut => Instance.TelemetryOptOut;
+        public static bool TelemetryOptedIn => Instance.TelemetryOptedIn;

```

Comment:
Just in case I'll mention (just to keep in mind, not to change anything in this PR):

I have a slight feeling of over-use of the concept TelemetryOptedIn/TelemetryOptedOut (and more than one variable is named `TelemetryOptedIn`). In some cases the concept (and variable name) TelemetryEnabled/TelemetryDisabled describes the reality more precisely. I'll clarify what I mean.

The telemetry (gathering) can be enabled or disabled.
It can be enabled either by default or as a result of the user action (opting-in).
In the same way it can be disabled either by default or as a result of the user action (opting-out).
For the telemetry functionality (whether to gather or not) it does not matter how the telemetry was enabled (was enabled by default or as a result of the user action), it only matters whether the telemetry is enabled or not (only the state matters, but not how that state has been achieved).

That is why I would recommend to consider using different terms (and variable names) 
for the _state_ (enabled or disabled)
and for the _user actions_ (opted-in or opted-out).

Reply:
Great point. I changed it to be `TelemetryEnabled`.